pkg/job_controller/api/v1/constants.go

```diff
@@ -45,6 +48,16 @@ const (
 	ResourceNvidiaGPU v1.ResourceName = "nvidia.com/gpu"
 )
 
+// add annonations
+const (
+	AnnotationPodBatchSize        = KubeDLPrefix + "/pod-batch-size"
+	AnnotationPodAffinityStrategy = KubeDLPrefix + "/pod-affinity-strategy"
```

Comment:
can you introduce a new field in `schedulePolicy` in job spec definition and enumerate values as `AllReduce, RingAllReduce...etc` instead of annotation?